.github/mergify.yml

```diff
@@ -0,0 +1,14 @@
+pull_request_rules:
+  - name: automatic merge for DiAGRAM
+    conditions:
+      - "head=combined-prs-branch"
+      - "check-success=Wiz Data Scanner"
+      - "check-success=Wiz IaC Scanner"
+      - "check-success=Wiz Secret Scanner"
+      - "check-success=Wiz Vulnerability Scanner"
+    actions:
+      review:
+        type: APPROVE
+        message: Automatically approving DiAGRAM PR
```

Comment:
Should this be `Automatically approving DiAGRAM PRs` or even `Automatically approving and merging DiAGRAM PRs`?